gene2phenotype_project/gene2phenotype_app/urls.py

```diff
@@ -24,323 +18,225 @@ def perform_create(self, serializer):
-        views.ListCurationEntries.as_view(),
-        name="list_curation_entries"
+        "curations/", views.ListCurationEntries.as_view(), name="list_curation_entries"
     ),
     path(
         "curation/<str:stable_id>/",
         views.CurationDataDetail.as_view(),
-        name="curation_details"
+        name="curation_details",
     ),
     path(
         "curation/<str:stable_id>/update/",
         views.UpdateCurationData.as_view(),
-        name="update_curation"
+        name="update_curation",
     ),
     path(
         "curation/<str:stable_id>/delete",
         views.DeleteCurationData.as_view(),
-        name="delete_curation"
+        name="delete_curation",
     ),
-
     ### Publish data ###
     path(
         "curation/publish/<str:stable_id>/",
         views.PublishRecord.as_view(),
-        name="publish_record"
+        name="publish_record",
     ),
-
     ### User management ###
+    path("create/user/", views.CreateUserView.as_view(), name="create_user"),
+    path("add_user/panel/", views.AddUserToPanelView.as_view(), name="add_user_panel"),
+    path("profile/", views.ManageUserView.as_view(), name="profile"),
     path(
-        "create/user/",
-        views.CreateUserView.as_view(),
-        name="create_user"
-    ),
-    path(
-        "add_user/panel/",
-        views.AddUserToPanelView.as_view(),
-        name="add_user_panel"
-    ),
-    path(
-        "profile/",
-        views.ManageUserView.as_view(),
-        name="profile"
-    ),
-    path(
-        "change_password/",
-        views.ChangePasswordView.as_view(),
-        name="change_password"
+        "change_password/", views.ChangePasswordView.as_view(), name="change_password"
     ),
     path(
         "reset_password/<uid>/<token>/",
         views.ResetPasswordView.as_view(),
-        name="reset_password"
+        name="reset_password",
     ),
+    path("verify/email/", views.VerifyEmailView.as_view(), name="verify_email"),
+    path("login/", views.LoginView.as_view(), name="_login"),
+    path("logout/", views.LogOutView.as_view(), name="logout"),
     path(
-        "verify/email/",
-        views.VerifyEmailView.as_view(),
-        name="verify_email"
-    ),
-    path(
-        "login/",
-        views.LoginView.as_view(),
-        name="_login"
+        "token/refresh/", views.CustomTokenRefreshView.as_view(), name="token_refresh"
     ),
+    ### Panels management ###
+    path("create/panel/", views.PanelCreateView.as_view(), name="panel_create"),
+    ### Meta information ###
+    path("reference_data/", views.MetaView.as_view(), name="get_reference_data"),
+    ### GenCC submission  ###
     path(
-        "logout/",
-        views.LogOutView.as_view(),
-        name="logout"
+        "gencc_create/", views.GenCCSubmissionCreateView.as_view(), name="create_gencc"
     ),
     path(
-        "token/refresh/",
-        views.CustomTokenRefreshView.as_view(),
-        name="token_refresh"
+        "unsubmitted-stable-ids/",
```

Comment:
To follow the same logic as the other endpoints, this should be called `unsubmitted_stable_ids/`